mythical-creatures/test/dragon-test.js

```diff
@@ -1,29 +1,31 @@
 var assert = require('chai').assert;
 var Dragon = require('../exercises/dragon');
 
-describe('Dragon', function () {
+describe('Dragon', function() {
 
-  it.skip('should be a function', function () {
+  it.skip('should be a function', function() {
     assert.isFunction(Dragon);
   });
 
-  it.skip('should instantiate our good friend, Dragon', function () {
+  it.skip('should instantiate our good friend, Dragon',  ()=> {
```

Comment:
Did you mean to change this into an arrow function?

Reply:
good catch